src/components/LilButton.vue

```diff
@@ -1,7 +1,8 @@
 <template>
   <button
+    class="lil-button"
```

Comment:
I dont think yhe parent class should use this to style the lil-button. We should discuss this